server/service/GithubService.js

```diff
@@ -146,21 +147,15 @@ export class GithubService {
   }
 
   readZapprFile(user, repo, accessToken) {
-    const repoContentUrl = API_URL_TEMPLATES.REPO_CONTENT
-                                            .replace('${owner}', user)
-                                            .replace('${repo}', repo)
-    const validZapprFileUrls = VALID_ZAPPR_FILE_PATHS.map(zapprFilePath => path.join(repoContentUrl, zapprFilePath))
-
-    const zapprFileRequests = validZapprFileUrls.map(zapprFileUrl => this.fetchPath('GET', zapprFileUrl, null, accessToken))
-    return promiseFirst(zapprFileRequests)
+    return this._readFile(VALID_ZAPPR_FILE_PATHS, user, repo, accessToken)
```

Comment:
Is the meaning of the `_` prefix "should not be exposed, do not use, internal API"?